dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -50,12 +53,20 @@
             <!-- 
                 DataCite (1)
                 Template Call for DOI identifier.
-            --> 
-            <xsl:apply-templates select="//dspace:field[@mdschema='dc' and @element='identifier' and starts-with(., 'http://dx.doi.org/')]" />
+                Occ: 1
+            -->
+            <!--
+                dc.identifier.uri may contain more than one DOI, e.g. if the
+                repository contains an item that is published by a publishing 
```

Comment:
helix84: That is not so simple as it might be. Let me point this out by just one example. A member of your university publishes a paper. The publishing company mints a DOI for the version containing the layout of the journal. The Open Access policies allows the author to publish the paper in the Open Access repository of its university as preprint, which means in the layout he submitted it to the publishing company. Further more the policies says that you must link to the original article using the DOI. As the layout differs, the page numbers differs. If you reuse the same DOI a citation of <doi>, p. 3 may cite to different parts depending whether you load the DOI from the publishing company or the Open Access version from the Open Access repository. This example is not a fictive one. Such policies are totally normal.

By the way: crossref is just one DOI registration agency. Others may have other rules. While I agree, that it would be great, if we could tell DSpace whether it should reuse a DOI or mint a new one, I don't think this is so simple and clear as one might think on the first view.